server/src/models/DFSPEndpointModel.js

```diff
@@ -0,0 +1,126 @@
+/******************************************************************************
+ *  Copyright 2019 ModusBox, Inc.                                             *
+ *                                                                            *
+ *  info@modusbox.com                                                         *
+ *                                                                            *
+ *  Licensed under the Apache License, Version 2.0 (the "License");           *
+ *  you may not use this file except in compliance with the License.          *
+ *  You may obtain a copy of the License at                                   *
+ *  http://www.apache.org/licenses/LICENSE-2.0                                *
+ *                                                                            *
+ *  Unless required by applicable law or agreed to in writing, software       *
+ *  distributed under the License is distributed on an "AS IS" BASIS,         *
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  *
+ *  See the License for the specific language governing permissions and       *
+ *  limitations under the License.                                            *
+ ******************************************************************************/
+
+'use strict';
+const { knex } = require('../db/database');
+const DFSPModel = require('./DFSPModel');
+const NotFoundError = require('../errors/NotFoundError');
+const InternalError = require('../errors/InternalError');
+const ENDPOINT_TABLE = 'dfsp_endpoint';
+
+/**
+ * Gets an raw endpoint by its id, and parses the JSON in value, returning an Object.
+ */
+exports.findById = async (id) => {
```

Comment:
that's the method for internal use, so no need to export it, also the naming can be more specific.
```suggestion
const findRowById = async (id) => {
```